src/models/createNewHouseModel.js

```diff
@@ -0,0 +1,103 @@
+const mongoose = require('mongoose')
+
+const housesSchema = new mongoose.Schema({
```

Comment:
You could consider naming the fields in the schema the same as they are called on the front-end to make saving and retrieving data as effortless as possible unless you think some fields on the front-end should change their names. You might possibly also reconsider embeddings in this schema to make them work best with the most common operations like reading basic house data to be displayed on the houses list and on the map.